backend/sass/common.blocks/_setup.scss

```diff
@@ -178,7 +176,7 @@ $kadena-orange: #F3AB3C;
 
 .setup__checkbox-wrapper {
   margin: 1rem;
-  .checkbox {
+  .setup__checkbox {
```

Comment:
Much better - the nesting in `.setup__checkbox-wrapper` should not be necessary though.